### DISCUSSION TITLE
What is the national dish of Hungary?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#125] [Galaxy Trivia] [answered]
**Hint:** soup / stew

**Comment by kjbaker-uk** [comment, marked as answer]
Goulash – Hungary's national dish (pronounced “gooyash,” by the way) is a cross between the two, a steaming bowl of slow-cooked beef, carrots, onions and loads of Hungary's trademark paprika to give it a good kick

**Reply by ParanoidUser** [reply]
Have you ever tried it? 😏

**Reply by kjbaker-uk** [reply]
I have not.
